assets/scss/core/_variables.scss

```diff
@@ -8,7 +8,8 @@ $dp-font-lineheight: 22px !default;
 $dp-font-small-lineheight: 18px;
 $dp-font-weight: 400 !default;
 $dp-font-family: Arial, sans-serif !default;
-$dp-font-family-proximanova: "proxima-nova", Helvetica;
+$dp-font-family-proximanova: "proxima-nova", Helvetica, Arial, sans-serif !default;
+$dp-font-family-handofsean: "hand-of-sean", Helvetica, Arial, sans-serif;
```

Comment:
Where are we using this font?

Reply:
For the moment the font Hand of Sean, no is used but is included in the library because in the sites actualy use.

Reply:
![font](https://user-images.githubusercontent.com/46735526/58899878-57753a80-86d4-11e9-9734-593dfb5aa9fd.gif)


Please check that the classes that are deleted, are not being used in the webapp

Reply:
If we are not using it in the library and we are not generating any class to reuse this font-family, why are we including it?

Reply:
Yes we are using this in zoho chat bubble.

Reply:
Ok!